insights/config/specs.py

```diff
@@ -246,6 +246,7 @@
     "ovirt_engine_server.log"   : SimpleFileSpec("var/log/ovirt-engine/server.log"),
     "ovs-vsctl_show"            : CommandSpec("/usr/bin/ovs-vsctl show"),
     "pacemaker.log"             : SimpleFileSpec("var/log/pacemaker.log"),
+    'pam.conf'                  : SimpleFileSpec("etc/pam.conf"),
```

Comment:
This doesn't seem related to the "Remove...". Better open a new PR for these specs adding.